rules/community/cloudwatch_events/cloudtrail_critical_api_calls.json

```diff
@@ -846,6 +846,81 @@
     "source": "prefix.cluster.sample.bucket",
     "trigger_rules": [
       "cloudtrail_critical_api_calls"
-    ]
+    ],
+    "publisher_tests": {
```

Comment:
I know I wrote this.... but 

Since you wrote the `lookuptable` and `threat_intel` sample rules, maybe should also split this into one of those `example_publisher_test` kind of rules? Up to you

Reply:
made update for this